docs/rancher/05-setup-lb/README.md

```diff
@@ -0,0 +1,169 @@
+
+Before beginning, ensure you have:
+
+- A server designated for the load balancer.
+- Basic understanding of Nginx configurations.
+- Access to a Kubernetes cluster via `kubectl`.
+
+## Step 1 - Preparing the Load Balancer Server
+
+Identify a server to serve as your load balancer. For instructions on setting up a server, see the [Ubuntu Server Setup Guide](../01-setup-ubuntu/README.md).
+
+## Step 2 - Installing Docker
+
+Docker streamlines the deployment and management of containerized applications. To install Docker on your load balancer server, follow these steps:
+
+First, update your package list:
+
+```bash
+sudo apt update
+```
+
+Then, install the necessary packages for HTTPS repository access:
+
+```bash
+sudo apt install apt-transport-https ca-certificates curl software-properties-common
+```
+
+Next, add the Docker repository GPG key:
+
+```bash
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
+```
+
+Include the Docker repository:
+
+```bash
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+```
+
+Update the package list again:
+
+```bash
+sudo apt update
+```
+
+Finally, install Docker:
+
+```bash
+sudo apt install docker-ce
+```
+
+Verify Docker installation by checking its status:
+
+```bash
+sudo systemctl status docker
+```
+
+Docker will now be installed and configured to start on boot.
+
+## Step 3 - Setting Up Nginx as a Load Balancer
+
+To configure Nginx as a load balancer, create a configuration file in `/etc/nginx.conf` to handle your traffic:
+
+First, open the configuration file for editing:
+
+```bash
+vim /etc/nginx.conf
+```
+
+Then, insert the following configuration, replacing `<server-address>` with the actual addresses of your services:
+
+```nginx
+worker_processes auto;
+worker_rlimit_nofile 40000;
+
+events {
+    worker_connections 8192;
+}
+
+stream {
+    upstream murmur_servers_http {
+        least_conn;
+        server <server-address-1>:80 max_fails=3 fail_timeout=5s;
+        server <server-address-2>:80 max_fails=3 fail_timeout=5s;
+        server <server-address-3>:80 max_fails=3 fail_timeout=5s;
+    }
+
+    upstream murmur_servers_https {
+        least_conn;
+        server <server-address-1>:443 max_fails=3 fail_timeout=5s;
+        server <server-address-2>:443 max_fails=3 fail_timeout=5s;
+        server <server-address-3>:443 max_fails=3 fail_timeout=5s;
+    }
+
+    server {
+        listen 80;
+        proxy_pass murmur_servers_http;
+        proxy_protocol on;
+    }
+
+    server {
+        listen 443;
+        proxy_pass murmur_servers_https;
+        proxy_protocol on;
+    }
+}
+```
+
+Note, if you only have one node for your RKE2 cluster, then you only need to have one server in murmur_servers_http and murmur_servers_https.
```

Comment:
It contradicts the content I read earlier: [For a HA setup, deploy an odd number of servers, at least **three**](https://github.com/MurmurationsNetwork/MurmurationsServices/tree/max/683/docs/rancher/04-setup-rke2-cluster#prerequisites), so I believe the one server mentioned here is not necessary. However, I believe it's important to tell user to adjust murmur_servers_http and murmur_servers_https based on the nodes they have.